src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,74 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
 public class WorkWithFile {
+    public static final int OPERATION_TYPE_COLOMN = 0;
+    public static final int AMMOUNT_COLOMN = 1;
```

Comment:
Do you use these constants outside this class?

Reply:
no, fixed